module/VuFind/src/VuFind/Auth/AbstractShibboleth.php

```diff
@@ -0,0 +1,362 @@
+<?php
+/**
+ * Shibboleth with WAYF authentication module.
```

Comment:
This comment (and the one above the class itself) should probably be updated to say "Abstract Shibboleth base class" or something similar.